test/gui/tst_vfs/test.feature

```diff
@@ -4,15 +4,23 @@ Feature: Enable/disable virtual file support
     I want to enable virtual file support
     So that I can synchronize virtual files with local folder
 
-    
+
     Scenario: Enable VFS
         Given user "Alice" has been created on the server with default attributes and without skeleton files
         And user "Alice" has set up a client with default settings
         When the user enables virtual file support
-        Then VFS enabled baseline image should match the default screenshot
+        Then the "Disable virtual file support..." button should be available
```

Comment:
It would be nice to include where the button is expected to be available.
```suggestion
        Then the "Disable virtual file support..." button should be available in (<some menu>or <location>)
```

Reply:
I think it is OK the way it is, the button might move somewhere else one day but the test-description should then still stay the same, just the implementation would change